### PR TITLE
fix old SA1006 lint errors

### PR DESCRIPTION
**Pull request by loadtheaccumulator**
# Description
Fixing simple SA1006 lint issues with formatted print statements and no formatting

FIXES: 

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [x] Refactor

### CODE REVIEW

File: pkg/models/thirdpartyrepo.go [hunk 1]
```diff
@@ -66,7 +65,7 @@ func (t *ThirdPartyRepo) ValidateRequest() error {
 		return errors.New(RepoNameCantBeInvalidMessage)
 	}
 	if !ValidateRepoURL(t.URL) {
-		return fmt.Errorf(InvalidURL)
+		return errors.New(InvalidURL)
```
Comment (ezr-ondrej) [inline pkg/models/thirdpartyrepo.go:68]: we could have the Error as a constant directly `InvalidURLErr = ....`. That way whoever consumes the package can use errors.Is() ... to tell which error was returned (if interested).
Not a big deal, we'd need to refactor all the errors here, which is probably not what we want to do.

Comment (ezr-ondrej) [reply pkg/models/thirdpartyrepo.go:68]: Would be a nice follow-up if you'd like :)

**Review by ezr-ondrej — APPROVED** [review]
Thank you! :+1:

File: pkg/models/thirdpartyrepo.go (hunk 1 above)
Comment (loadtheaccumulator) [reply pkg/models/thirdpartyrepo.go:68]: Definitely. Constants are used elsewhere, so we can always add error refactor to the list.

**Pull request merged**